core/Cargo.toml

```diff
@@ -295,7 +295,7 @@ suppaftp = { version = "5.2", default-features = false, features = [
   "async-rustls",
   "async-native-tls",
 ], optional = true }
-tikv-client = { version = "0.2.0", optional = true, default-features = false }
+tikv-client = { version = "0.3.0", optional = true, default-features = false }
```

Comment:
Is it necessary?

Reply:
Yes, tikv-client 0.2 will require openssl.

Reply:
Thanks for you information!